some broken BIOSes offer 24bit depth with 8 bits per pixel mode
this is just wrong, so ignore all modes below (and including) 8 bits.

--- linux-2.6.16.orig/drivers/video/vesafb-tng.c	Tue May  2 20:06:30 2006
+++ linux-2.6.16/drivers/video/vesafb-tng.c	Tue May  2 20:06:56 2006
@@ -781,7 +781,7 @@
 		 * hardware configuration (D0), color (D3), graphics (D4)
 		 * and that have support for the LFB (D7). */
 		if ((mib->mode_attr & 0x99) == 0x99 &&
-		    mib->bits_per_pixel >= 8) {
+		    mib->bits_per_pixel > 8) {
 			off++;
 		} else {
 			vbe_modes_cnt--;
